staging/src/k8s.io/kubectl/pkg/cmd/drain/drain.go

```diff
@@ -194,6 +195,7 @@ func NewCmdDrain(f cmdutil.Factory, ioStreams genericclioptions.IOStreams) *cobr
 	cmd.Flags().StringVarP(&o.drainer.PodSelector, "pod-selector", "", o.drainer.PodSelector, "Label selector to filter pods on the node")
 	cmd.Flags().BoolVar(&o.drainer.DisableEviction, "disable-eviction", o.drainer.DisableEviction, "Force drain to use delete, even if eviction is supported. This will bypass checking PodDisruptionBudgets, use with caution.")
 	cmd.Flags().IntVar(&o.drainer.SkipWaitForDeleteTimeoutSeconds, "skip-wait-for-delete-timeout", o.drainer.SkipWaitForDeleteTimeoutSeconds, "If pod DeletionTimestamp older than N seconds, skip waiting for the pod.  Seconds must be greater than 0 to skip.")
+	cmd.Flags().BoolVar(&o.drainer.ParallelizeNodes, "parallelize-nodes", o.drainer.ParallelizeNodes, "Execute drains on multiple nodes in parallel")
```

Comment:
I don't like this name that much, maybe just `--parallel`? Other option is to have something like `--num-workers` which would allow you to specify the number of parallel drains, wdyt?

Reply:
Cool, I've updated the flag to be `--parallel`